data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/encryption/request/DecryptRpcCommand.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "DecryptRpcCommand",
+  "namespace": "net.corda.data.crypto.wire.ops.encryption.request",
+  "doc": "Request to decrypt the given byte array",
+  "fields": [
+    {
+      "name": "category",
+      "type": "string",
+      "doc": "The category of HSM e.g. ENCRYPTION_SECRET, TLS, etc."
+    },
+    {
+      "name": "alias",
+      "type": ["null", "string"],
+      "doc": "The symmetric key alias."
+    },
+    {
+      "name": "cipherBytes",
+      "type": "bytes",
+      "doc": "The data to decrypt."
+    },
+    {
+      "name": "context",
```

Comment:
I think adding this `context` was was decided at design level. 

However, I have seen before cases where we have been using it while we shouldn't and we had to go in there and revert all of that, to use some other field instead back then (IRRC we used to pass the signature spec in context. This piece of work can be found [here](https://r3-cev.atlassian.net/browse/CORE-11521)). 

I would say if you are not planning on using it remove it as it will cause at the very least confusion on the API, like what is good for. It can always be added later if found it's needed which I doubt.

Reply:
I prefer to remove it. 

@dickon WDYT?